package.json

```diff
@@ -12,6 +12,7 @@
     "@types/chrome": "0.0.284",
     "@types/dompurify": "3.2.0",
     "@types/jquery": "3.5.32",
+    "@types/linkifyjs": "^2.1.7",
```

Comment:
here we don't need separate types package, as starting from v3 `linkifyjs` includes ts types in the main package - https://github.com/nfrasser/linkifyjs/releases/tag/v3.0.0, they're available at `node_modules/linkifyjs/dist/` or `node_modules/linkify-html/dist/`

Reply:
Yep. you are right!
Fixed